### DISCUSSION TITLE
Swift/Kotlin/Elixir language highlighting?

### DISCUSSION THREAD

**Discussion by spotlightishere** [outline/outline#3890] [Ideas]
Hi, all! I started using Outline a few days ago, and am loving it so far!

I'd like to propose adding three languages to have highlighting: Swift, Kotlin, and Elixir.

These languages both appear higher up on the [2022 Stack Overflow "Most Loved" languages list](https://survey.stackoverflow.co/2022/#technology-most-loved-dreaded-and-wanted), and have a high amount of users. Additionally, highlighting already supports Objective-C (for Swift) and Java (for Kotlin) respectively, so this may make it easier for such developers.

It appears that [Refractor](https://github.com/wooorm/refractor#data) already supports all three.
Testing locally, only three changes would seem to be changed: adding their imports & adding to the registration array in `shared/editor/nodes/CodeFence.ts`, and then adding to the `LANGUAGES` object in `shared/editor/plugins/Prism.ts`.

Thoughts on adding these languages? Would any other areas need editing to support?
Thank you!

**Comment by tommoor** [comment]
Sounds good, go for it!
